modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/rest/TryItService.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ * Copyright (c) 2016-2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```

Comment:
This should be just 2016

Reply:
Fixed in https://github.com/ballerinalang/composer/pull/4354